### PR TITLE
StartScreen work

### PR DESCRIPTION
**Pull request by gmartin7**
* add React-Navigation deps, but haven't used them yet
   (It builds! That's an important step!)
* get the basic StartScreen look like it should

### CODE REVIEW

File: app.json [hunk 1]
```diff
@@ -3,11 +3,10 @@
     "name": "bloompub-viewer-RN",
     "slug": "bloompub-viewer-RN",
     "version": "1.0.0",
-    "orientation": "portrait",
     "icon": "./assets/icon.png",
     "userInterfaceStyle": "light",
     "splash": {
-      "image": "./assets/splash.png",
+      "image": "./assets/icon.png",
```
Comment (jsubloom) [inline app.json:9]: probably ok to delete any unused starter icons now?

File: src/StartScreen.tsx [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+    const goRead = () => {}
+
+    return (
+        <View style={styles.startScreen}>
+            <Image
+                style={styles.logo}
+                source={require('../assets/wordmark.png')}
+            />
+            <TouchableHighlight
+                onPress={goRead}
```
Comment (jsubloom) [inline src/StartScreen.tsx:14]: Ok, so i guess in the future we'll make clicking the button call navigate() to a Book Read screen or something?

This is fine for now.

File: src/StartScreen.tsx [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+                style={styles.highlight}
+            >
+                <View style={styles.buttonContainer}>
+                    <Image
+                        style={styles.button}
+                        source={require('../assets/Open.png')}
+                    />
+                    <Text style={styles.text}>Choose BloomPUB book on this computer</Text>
```
Comment (jsubloom) [inline src/StartScreen.tsx]: Maybe "device" or "phone" instead of "computer" now?

**Review by jsubloom — CHANGES_REQUESTED** [review]
yay nice, the start screen is coming together!

File: app.json (hunk 1 above)
Comment (gmartin7) [reply app.json:9]: Done

File: src/StartScreen.tsx (hunk 2 above)
Comment (gmartin7) [reply src/StartScreen.tsx:14]: 👍 exactly

File: src/StartScreen.tsx (hunk 3 above)
Comment (gmartin7) [reply src/StartScreen.tsx]: I went with 'device' for now, since they could be using an ipad or something.

**Pull request merged**